libdnf.spec

```diff
@@ -294,6 +295,25 @@ Ruby bindings for the libdnf-cli library.
 %endif
 
 
+# ========== libdnf-plugin-python-plugins-loader ==========
+
+%if %{with python_plugins_loader}
+%package -n python3-libdnf-python-plugins-loader
+Summary:        Libdnf plugin that allows loading Python plugins
+License:        LGPLv2.1+
```

Comment:
`LGPLv2.1+` -> `LGPLv2+`

Reply:
I copied the license header from another libdnf source file -> We need to check all the files.

Reply:
Yes, there's a problem in licenses in other files too.
I'll take care of them in another PR or push the changes directly.